### DISCUSSION TITLE
Radeon 5850 is slow in 10.15.7 - Apple Software Renderer is used

### DISCUSSION THREAD

**Discussion by mrdc** [chris1111/Legacy-Video-patch#22] [General]
Hello,

I’m trying to make 5850 a little bit faster in 10.15.7 - in 10.13 this card works fine - acceleration etc,
10.15.7 uses Apple Software Renderer and GUI is slow :/

![974D3A8A-B1C2-46B6-950B-6A028D556C9D](https://user-images.githubusercontent.com/16876990/107614780-06189f00-6c5c-11eb-9ad5-fbd39c98be7e.png)
![B1AD8BD3-B3A5-4799-B735-ADD0533FE495](https://user-images.githubusercontent.com/16876990/107614783-07e26280-6c5c-11eb-8bc7-22eed394029a.png)

**Comment by chris1111** [comment]
I know

**Reply by mrdc** [reply]
So, it’s correct behavior?
Then I’ll replace video card.
